source/develop/dev-process/working-with-gerrit.md

```diff
@@ -230,23 +234,15 @@ The patch life cycle process comprises of the following steps:
 
 ### Submitting a Topic Branch to Gerrit
 
-A [topic branch](http://progit.org/book/ch3-4.html) is a short-lived branch that you create and use for a single particular feature or related work.
+A [feature branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) is a short-lived branch that you create and use for a single particular feature or related work.
 
-*   First, download the git-review tool from Openstack and copy it to your project
-    -   git-review is a tool that helps submitting git branches to Gerrit for review.
-    -   URL: <https://github.com/openstack-infra/git-review>
+#### Git Review
+git-review is a tool that helps submitting git branches to Gerrit for review.
 
-&nbsp;
-
-        $ git clone https://github.com/openstack-infra/git-review
-        $ cp git-review/git-review project/
```

Comment:
Not sure about the removal of this. I don't use git review and better ask someone who does.
The git link did change to https://opendev.org/opendev/git-review

Reply:
Can you add to review someone that uses git review?

Reply:
not sure if someone is actually using it..

Reply:
can maybe someone from the OpenStack team review it? :)

Reply:
@nirs IIRC you're using git review